esrally/track/params.py

```diff
@@ -732,6 +738,48 @@ def percent_completed(self):
         return self.current_bulk / self.total_bulks
 
 
+class OpenPointInTimeParamSource(ParamSource):
+    def __init__(self, track, params, **kwargs):
+        super().__init__(track, params, **kwargs)
+        if len(track.indices) == 1:
+            default_target = track.indices[0].name
+        elif len(track.data_streams) == 1:
+            default_target = track.data_streams[0].name
+        else:
+            default_target = None
+        # indices are preferred but data streams can also be queried the same way
+        target_name = params.get("index")
+        type_name = params.get("type")
+        if not target_name:
+            target_name = params.get("data-stream", default_target)
```

Comment:
is `data-stream` intentionally undocumented for `search`? also undocumented here